frontend/src/components/Errors/ErrorNotCollaborator.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import GoToTripsButton from './GoToTripsButton.js';
+
+/**
+ * Error component displayed for when the current user is not listed as a
+ * collaborator for the trip they are trying to access.
+ */
+class ErrorNotCollaborator extends React.Component {
+  /** @inheritdoc */
+  render() {
+    return (
+      <div>
+        <p>Sorry, you're not authorized to view this trip.</p>
+        <GoToTripsButton />
+      </div>
+    );
+  }
+}
+
+export default ErrorNotCollaborator;
```

Comment:
I mentioned this on another PR, but it may be better to just use the trip not found page for this so that no information is potentially leaked to unauthorized 3rd parties (knowing a trip exists is more than knowing nothing).

Reply:
Sounds good, I have the ErrorNotCollaborator component deleted and its logic now handled by ErrorTripNotFound.